diceRoll.py

```diff
@@ -31,10 +39,24 @@ def dicegame(total_players,maximum_points):
 				break
 			if a_scores>cpu:
 				print("You are leading at the moment!")
+				print("You are leading at the moment!")
```

Comment:
#Comments inclusion

##Since we have already introduced comments to explain the current state, inclusion of multiple comments seems to be redundant. Kindly revisit the flow to ensure if the additional comments are required.